include/debug.h

```diff
@@ -357,9 +357,9 @@ static inline const char *colorfilter(const char *x) {
   do {                     \
                            \
     if (res < 0)           \
-      PFATAL(x);           \
+      ABORT(x);           \
     else                   \
-      FATAL(x);            \
+      ABORT(x);            \
```

Comment:
is it necessary to change this? or can you use your own function that does not change the behaviour of afl++?

Reply:
I can't remember exactly why I changed this. From what I recalled, this change probably related to debugging. I will confirm this.